layers/+themes/themes-megapack/packages.el

```diff
@@ -37,7 +37,8 @@
     dracula-theme
     espresso-theme
     farmhouse-theme
-    firebelly-theme
+    ;; Removed from melpa
+    ; firebelly-theme
```

Comment:
I only looked at the changes, so didn't see the rest of the file :-( Anyway, lets just leave it as it is (single semicolon to disable a theme)